modules/logging-architecture-overview.adoc

```diff
@@ -6,24 +6,30 @@
 [id="logging-architecture-overview_{context}"]
 = Logging architecture
 
-The {logging} consists of these logical components:
-
-* `Collector` - Reads container log data from each node and forwards log data to configured outputs.
-
-* `Store` - Stores log data for analysis; the default output for the forwarder.
-
-* `Visualization` - Graphical interface for searching, querying, and viewing stored logs.
-
-These components are managed by Operators and Custom Resource (CR) YAML files.
+The major components of the {logging} are:
+
+Collector:: The collector is a daemonset that deploys pods to each {product-title} node. It collects log data from each node, transforms the data, and forwards it to configured outputs. You can use the Vector collector or the legacy Fluentd collector.
++
+--
+include::snippets/logging-fluentd-dep-snip.adoc[]
+--
+
+Log store:: The log store stores log data for analysis and is the default output for the log forwarder. You can use the default LokiStack log store, the legacy Elasticsearch log store, or forward logs to additional external log stores.
++
+--
+include::snippets/logging-elastic-dep-snip.adoc[]
+--
+
+Visualization:: You can use a UI component to view a visual representation of your log data. The UI provides a graphical interface to search, query, and view stored logs. The {product-title} web console UI is provided by enabling the {product-title} console plugin, but you can also use Kibana.
```

Comment:
you have to use Kibana  to show data on elasticsearch. You can not use kibana for Lokistack.

When lokistack is the default storage, the {product-title} web console UI is provided by enabling the {product-title} console plugin
When elasticsearch is used as default storage,  You have to use Kibana.